web/pandas/config.yml

```diff
@@ -146,16 +146,6 @@ sponsors:
     url: https://numfocus.org/
     logo: static/img/partners/numfocus.svg
     kind: numfocus
-  - name: "Two Sigma"
-    url: https://www.twosigma.com/
-    logo: static/img/partners/two_sigma.svg
```

Comment:
I suppose we can remove these assets too from the repo?

Reply:
You are right, I didn't realize. Deleted them now, thank you!